javascript/src/main/components/OfficeHours/OfficeHourTable.js

```diff
@@ -37,14 +37,14 @@ export default ({officeHours,admin,deleteOfficeHour}) => {
 
     if (admin) {
         columns.push({
-            text: "Delete",
+            text:  "Delete",
```

Comment:
Everything looks good. Maybe delete the white space in line 40 to reduce the amount of changes made